### PR TITLE
kakounePlugins.kak-idris: init at 2020-12-29

### PR DESCRIPTION
**Pull request by malte-v**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/kakoune/plugins/kak-idris.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, idris2, nodejs }:
+stdenv.mkDerivation {
+  name = "kak-idris";
+  version = "2020-12-29";
```
Comment (hrhino) [inline pkgs/applications/editors/kakoune/plugins/kak-idris.nix:4]: I think `unstable-` prefix is used for non-released versions

File: pkgs/applications/editors/kakoune/plugins/kak-idris.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  src = fetchFromGitHub {
+    owner = "stoand";
+    repo = "kakoune-idris";
+    rev = "1acdfb5d89e3951ae4bdf4a5fa2377b36448083d";
+    sha256 = "06qny8790j9d7vjs0pfyw71xvc6hmkjl8ssi483mbwbxs0zv6j9r";
+  };
+
+  patchPhase = ''
+    sed -i 's \bnode\b ${nodejs}/bin/node g' idris.kak
```
Comment (hrhino) [inline pkgs/applications/editors/kakoune/plugins/kak-idris.nix:14]: apparently `|` is preferred as the alternative sed-separator if `/` is unworkable...

**Review by hrhino — CHANGES_REQUESTED** [review]
So, there seems to be a file `pkgs/applications/editors/kakoune/plugins/generated.nix` containing such information, and it's made from a sibling `kakoune-plugin-names` file; the comment in `default.nix` implies that you can add the `patchPhase` via `overrides.nix`, as is done for several other packages.

Could this be adapted to use that model?

**Comment by malte-v** [comment]
I'm not interested in merging this anymore. If you want to have this merged, please open a new PR and make yourself the maintainer.

**Pull request closed without merging**